resources/bixby-watch/layout-macros/Movie_Details_Watch.layout.bxb

```diff
@@ -0,0 +1,145 @@
+layout-macro-def (MOVIE_DETAILS_WATCH) {
+	params {
+		param (movie) {
+			type (Movie)
+			min (Required)
+			max (One)
+		}
+	}
+	content {
+		layout {
+			section {
+				content {
+					paragraph {
+						value ("#{value(movie.title)}")
+						style (Title_M)
+					}
+					spacer
+					divider
+					spacer
+					hbox {
+						content {
+							vbox {
+								grow (1)
+								halign (Center)
+								content {
+									text {
+										value ("Released")
+										style (Detail_L_Soft)
+									}
+									text {
+										value ("#{dateTime(movie.releaseDate, 'MMM dd, yyyy')}")
+										style (Title_S)
+									}
+								}
+							}
+						}
+					}
```

Comment:
I would get rid of these `hbox` components and just use paragraph/single line. The `hbox` doesn't do anything here, since there's only 1 `vbox`. Text on the watch is automatically centered for all components unless otherwise specified. 

Everything else looks good

Reply:
Made the change!